shared/constants/types/rpc-gen.tsx

```diff
@@ -0,0 +1,4107 @@
+export type ConfigValue = {readonly isNull: Boolean; readonly b?: Boolean | null; readonly i?: Int | null; readonly s?: String | null; readonly o?: String | null}
+export type ConfiguredAccount = {readonly username: String; readonly hasStoredSecret: Boolean}
+export type ConfirmResult = {readonly identityConfirmed: Boolean; readonly remoteConfirmed: Boolean; readonly expiringLocal: Boolean; readonly autoConfirmed: Boolean}
+export type ConflictAutomaticResolving = {readonly isStuck: Boolean}
+export type ConflictGeneration = Int
+export type ConflictManualResolvingLocalView = {readonly serverView: Path}
+export type ConflictManualResolvingServerView = {readonly localViews?: Array<Path> | null}
+export type ConflictState = {conflictStateType: 0; automaticresolving: ConflictAutomaticResolving | null} | {conflictStateType: 1; manualresolvingserverview: ConflictManualResolvingServerView | null} | {conflictStateType: 2; manualresolvinglocalview: ConflictManualResolvingLocalView | null}
+export type ConflictStateType =
+  | 0 // AutomaticResolving_0
+  | 1 // ManualResolvingServerView_1
+  | 2 // ManualResolvingLocalView_2
+
+export type Contact = {readonly name: String; readonly components?: Array<ContactComponent> | null}
+export type ContactComponent = {readonly label: String; readonly phoneNumber?: RawPhoneNumber | null; readonly email?: EmailAddress | null}
+export type CopyArgs = {readonly opID: OpID; readonly src: Path; readonly dest: Path}
+export type CryptKey = {readonly KeyGeneration: Int; readonly Key: Bytes32}
+export type Cryptocurrency = {readonly rowId: Int; readonly pkhash: Bytes; readonly address: String; readonly sigID: SigID; readonly type: String; readonly family: String}
+export type CsrfToken = String
+export type CurrentStatus = {readonly configured: Boolean; readonly registered: Boolean; readonly loggedIn: Boolean; readonly sessionIsValid: Boolean; readonly user?: User | null}
+export type DbKey = {readonly dbType: DbType; readonly objType: Int; readonly key: String}
+export type DbType =
+  | 0 // MAIN_0
+  | 1 // CHAT_1
+
+export type DbValue = Bytes
+export type DeletedTeamInfo = {readonly teamName: String; readonly deletedBy: String; readonly id: Gregor1.MsgID}
+export type DesktopStatus = {readonly version: String; readonly running: Boolean; readonly log: String}
+export type Device = {readonly type: String; readonly name: String; readonly deviceID: DeviceID; readonly cTime: Time; readonly mTime: Time; readonly lastUsedTime: Time; readonly encryptKey: KID; readonly verifyKey: KID; readonly status: Int}
+export type DeviceDetail = {readonly device: Device; readonly eldest: Boolean; readonly provisioner?: Device | null; readonly provisionedAt?: Time | null; readonly revokedAt?: Time | null; readonly revokedBy: KID; readonly revokedByDevice?: Device | null; readonly currentDevice: Boolean}
+export type DeviceEk = {readonly seed: Bytes32; readonly metadata: DeviceEkMetadata}
+export type DeviceEkMetadata = {readon
```

Comment:
spoke IRL but i think the enum and this type of type can be merged into one thing now